api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,21 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
This is a generated file and also this comment should stay, please revert this file changes.

Reply:
woops, this has been fixed @TheoBrigitte :bow: